tests/bento_service_examples/coreml_classifier.py

```diff
@@ -0,0 +1,18 @@
+import numpy
+import pandas as pd
+from coremltools.models import MLModel  # pylint: disable=import-error
```

Comment:
I can reproduce this recursion error on my local machine in docker(linux environment with a similar setup with conda, py36 and ran the setup script in this PR). The error went away after I removed this import here and the type annotation below. 

Not sure why that is the case, I will look a bit more into it.

Reply:
I also removed the `predict` call in the test, so I can test the `save` under non-mac enviornment

Reply:
@guy4261 hope you don't mind, I will try pushing some changes to your branch to trigger some test runs here

Reply:
Update: just tried running the quick start guide example and it also fails in this environment. I guess it has to do with the python import path and how BentoML is installed.

Reply:
Hi @parano , I see the unit tests passed (on Travis). So is the code broken or not? I got confused 😰